schema/odata.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Are any of these properties required?

For integer types i think you should not use minLength but for example:

"minimum": 0,

Consider setting default value for boolean type fields (if you think its needed)

Reply:
Oh no 😠...
It seems I made that `asyncapi` branch from the `odata` in #417

Reply:
would it be easier then to do them together in this one ?

Reply:
Yeah, maybe, if that's not a problem for anyone

Reply:
not from my side, we are expecting the async.json file, right? :sweat_smile: